### PR TITLE
workspace & booking model

### PR DESCRIPTION
**Pull request by ericblandet**

### CODE REVIEW

File: app/models/workspace.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Workspace < ApplicationRecord
+  belongs_to :user
+  validates :name, :description, :address, :capacity, :daily_rate, presence: true
+  validates :daily_rate, numericality: { greater_than: 0 }
+  validates :capacity, numericality: { only_integer: true }
+end
```
Comment (Simon-Busch) [inline app/models/workspace.rb:8]: has_many: bookings ?

File: app/views/workspaces/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<h1><%= @workspace.name %></h1>
+
+<p><i class="fal fa-user-clock"></i> <%= @workspace.capacity %></p>
```
Comment (Simon-Busch) [inline app/views/workspaces/show.html.erb]: Add the @workspace.address

File: db/schema.rb [hunk 3]
```diff
@@ -10,11 +10,23 @@
...
 
-ActiveRecord::Schema.define(version: 2020_11_23_135613) do
+ActiveRecord::Schema.define(version: 2020_11_23_153707) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "bookings", force: :cascade do |t|
+    t.bigint "users_id", null: false
+    t.bigint "workspaces_id", null: false
```
Comment (Simon-Busch) [inline db/schema.rb:20]: What is that ?

**Review by Simon-Busch — APPROVED** [review]

File: db/schema.rb (hunk 3 above)
Comment (ericblandet) [reply db/schema.rb:20]: automatically generated

File: app/views/workspaces/show.html.erb (hunk 2 above)
Comment (ericblandet) [reply app/views/workspaces/show.html.erb]: ok!

**Review by Simon-Busch — APPROVED** [review]
Ok for me

**Pull request merged**